gatsby-browser.js

```diff
@@ -71,6 +71,13 @@ function applyZoomEffect({ excludedSelector, ...options }) {
         el.removeEventListener("load", onImageLoad)
       }
       el.addEventListener("load", onImageLoad)
+      el.setAttribute('tabIndex', 0)
+      el.addEventListener('keydown', (e) => {
```

Comment:
How about removing also? (in [line 72](https://github.com/JaeYeopHan/gatsby-remark-images-medium-zoom/pull/7/files#diff-3863fe9c963d5c58d29a7f5af77e6480R72))

Reply:
@crutchcorn
oh, my mistake. sorry... Please ignore this comment.